bentoml/_internal/cli/model_management.py

```diff
@@ -150,7 +150,22 @@ def delete(
         * Bulk delete all models with a specific name, e.g.: `bentoml model delete IrisClassifier`
         * Bulk delete multiple models by name and version, separated by ",", e.g.: `benotml model delete Irisclassifier:v1,MyPredictService:v2`
         """  # noqa
-        pass
+
+        def delete_target(target):
+            to_delete_models = model_store.list(target)
+
+            for model in to_delete_models:
+                if yes:
+                    if_delete = True
+                else:
+                    if_delete = click.confirm(f"delete bento {model.tag}?")
+
+                if if_delete:
+                    click.echo(f"deleting bento {model.tag}")
```

Comment:
similarly to above, I'd suggest remove the `deleting` line, and add a `deleted` message after the deletion call.